src/shared/tpm2-util.c

```diff
@@ -4483,6 +4483,7 @@ static const char* tpm2_userspace_event_type_table[_TPM2_USERSPACE_EVENT_TYPE_MA
 
 DEFINE_STRING_TABLE_LOOKUP(tpm2_userspace_event_type, Tpm2UserspaceEventType);
 
+#if HAVE_OPENSSL
 const char *tpm2_userspace_log_path(void) {
```

Comment:
This function is exported, but the declaration in the header is not ifdef'ed, which will probably generate compiler warnings. It's not even used outside this module, so just make it static and remove it from the header

Reply:
or maybe just move the ifdef below this block

Reply:
I'm fine with either. If you'd like me to update the patch just pick one, or do it yourself whatever way you'd like.